examples/overlays/overlay.yaml

```diff
@@ -0,0 +1,8 @@
+overlays: 1.0.0
+extends: ./openapi.yaml
```

Comment:
do we have an explanation in our docs that extends must be in the form of absolute/relative path i.e. .openapi.yaml won't work but ./openapi.yaml will work?

I see it as a potential problem